microdnf/context.cpp

```diff
@@ -44,6 +44,15 @@ namespace fs = std::filesystem;
 namespace microdnf {
 
 bool userconfirm(libdnf::ConfigMain & config) {
+    for (const auto & tsflag : config.tsflags().get_value()) {
+        if (tsflag == "test") {
+            std::cout << "Test mode enabled: Only package downloads, gpg key installations and transaction checks will "
+                         "be performed."
+                      << std::endl;
+            break;
+        }
+    }
+
```

Comment:
Strictly speaking this is not a user confirmation, looking at this I wonder if we should re-structure the methods, but we can leave it for now.